_posts/2017-05-02-newsletter.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: newsletter
+title: "Newsletter #8 - Open up the Windows"
+category: newsletter
+permalink: /news/2017/05/
+---
+
+What is Neovim?
+---------------
+
+Neovim is a fork of the venerable text-editor vim, focused on extensibility and
+usability. It is not a rewrite but a continuation and extension of Vim. Many
+clones and derivatives exist, some very clever—but none are Vim. Neovim is built
+for users who want the good parts of Vim, and more. See a list of differences
+via [:help vim-differences](https://neovim.io/doc/user/vim_diff.html).
+
+### Participating
+
+If you are interested in contributing to neovim, visit [the github
```

Comment:
Should not “Neovim” start with a capital here and in some places below?

Reply:
Right, thanks!